Updates to Android Studio and IntelliJ opening and new Flutter project instructions (#8711)
flutter/website: src/tools/android-studio.md

### Creating a new project

> The steps to create a new Flutter project from the Flutter starter app template are 
> slightly different for Android Studio and IntelliJ.

Reviewer: **nit:** Write as shorter sentences per [GSG](https://developers.google.com/style/accessibility#ease-of-reading).

```suggestion
Creating a new Flutter project from the Flutter starter app template
differs between Android Studio and IntelliJ.
```
